src/dialog/dialog.scss

```diff
@@ -0,0 +1,20 @@
+.Dialog {
```

Comment:
Copying here the [comment](https://github.com/primer/css/pull/1933#discussion_r801198200) from @langermank:

> @vdepizzol should we try and manage all component specific cases as props within overlay.scss or allow component-specific CSS on top?